src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -838,4 +814,10 @@
   <data name="ColorScheme_RenameErrorTip.Title" xml:space="preserve">
     <value>This color scheme name is already in use.</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="SettingContainer_OverrideIntro.Text" xml:space="preserve">
+    <value>Overrides a setting</value>
+  </data>
+  <data name="SettingContainer_ResetButtonHelpText" xml:space="preserve">
+    <value>Reset</value>

```

Comment:
why is this a HelpText and not a `[using:blah blah blah]tooltip`?

Reply:
Does this need to be a `.HelpText`?

Reply:
nah HelpText is a magic thing we made up.

Reply:
"HelpText" was just the name I happened to use here. No relationship to the `HelpText` property.

Going back to today's episode of "Carlos' issues with data binding", this is another case of that. I want to create one resw entry that can be used as the tooltip and the automation property for this button simultaneously.

I tried this (after setting the resw entry to `SettingContainer_ResetButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip`):
```xml
<Button x:Uid="SettingContainer_ResetButton"
		x:Name="ResetButton"
        Style="{StaticResource SettingContainerResetButtonStyle}"
        AutomationProperties.Name="{Binding RelativeSource={RelativeSource Self}, Path=ToolTip, Mode=OneWay}">
```

The tooltip is set properly, but the automation property is not.

Figured it'd be more straightforward to do this in the code like this:
```cpp
// SettingContainer.cpp --> OnApplyTemplate()

if (auto child{ GetTemplateChild(L"ResetButton") })
{
	// apply tooltip and name (automation property)
	const auto& helpText{ RS_(L"SettingContainer_ResetButtonHelpText") };
	Controls::ToolTipService::SetToolTip(child, box_value(helpText));
	Automation::AutomationProperties::SetName(child, helpText);
}
```

Reply:
Spoke with somebody from the accessibility team. Decided on this solution:
- Header (name of the setting) --> AutomationProperty::Name
- Override Message (overriddes what?) --> AutomationProperty::HelpText
- ToolTip (extra description to setting) --> AutomationProperty::DescriptionText